ui/helpers/utils/collectibles.js

```diff
@@ -0,0 +1,3 @@
+export const getCollectibleImageAlt = ({ name, tokenId, description }) => {
+  return description ?? `${name} ${tokenId}`;
```

Comment:
Also i think we should have a callback if there's no alt text provided, wdyt? like empty string